operator/pkg/validate/validate.go

```diff
@@ -18,15 +18,22 @@ import (
 	"fmt"
 	"reflect"
 
+	"github.com/ghodss/yaml"
 	"istio.io/api/operator/v1alpha1"
+
```

Comment:
reformat

Reply:
Actually I might have not run `make format` . Oops